File: runtimes/go/time.go
package main

import (
	"fmt"
	"time"
	"math"
	"strings"
	"reflect"
	"unsafe"
)

${SOLUTION}

${UTILITIES}

func main() {
    cur := time.Now().UnixNano() / 1_000_000
    ${CALL_SOLUTION}
    now := time.Now().UnixNano() / 1_000_000
    fmt.Print(now - cur)
}

File: runtimes/rust/time.rs
pub struct Solution;

use std::time::Instant;
use std::rc::Rc;
use std::cell::RefCell;
use std::collections::VecDeque;

${IMPORTS}

fn main() {
    let start = Instant::now();
    let _ = ${CALL_SOLUTION};
    let duration = start.elapsed();
    println!("{}", duration.as_millis());
}

${SOLUTION}

${UTILITIES}
